src/BCrypt/BCrypt.Helpers.cs

```diff
@@ -13,6 +13,11 @@ namespace PInvoke
     public static partial class BCrypt
     {
         /// <summary>
+        /// An array whose content doesn't matter.
+        /// </summary>
+        private static readonly byte[] OneElementDummyArray = new byte[1];
```

Comment:
Why does it need to be a one element array instead of zero element array ? (Not that it really matter for a private thing nobody is supposed to write to, i'm just curious)